mlir/utils/jenkins/Jenkinsfile

```diff
@@ -815,15 +824,21 @@ pipeline {
                         steps {
                             sh 'rm -rf composable_kernel'
                             dir('composable_kernel') {
-                                catchError (buildResult: null) {
-                                    getAndBuildCK('''
-                                        -DGPU_TARGETS="gfx1030;gfx908;gfx90a"
-                                        -DCMAKE_CXX_FLAGS="-O3"
-                                        -DCMAKE_PREFIX_PATH="/opt/rocm"
-                                        -DCMAKE_INSTALL_PREFIX=${WORKSPACE}/composable_kernel/build/CKInstallDir
-                                        -DCMAKE_BUILD_TYPE=Release
-                                        ''')
-                                    sh 'cd build/library; make install; cp ../composable_kernelConfig*.cmake ${WORKSPACE}/composable_kernel/build/CKInstallDir/lib/cmake/composable_kernel'
+                                script{
+                                    try {
+                                        getAndBuildCK('''
+                                            -DGPU_TARGETS="gfx1030;gfx908;gfx90a"
+                                            -DCMAKE_CXX_FLAGS="-O3"
+                                            -DCMAKE_PREFIX_PATH="/opt/rocm"
+                                            -DCMAKE_INSTALL_PREFIX=${WORKSPACE}/composable_kernel/build/CKInstallDir
+                                            -DCMAKE_BUILD_TYPE=Release
+                                            ''')
+                                        sh 'cd build/library; make install; cp ../composable_kernelConfig*.cmake ${WORKSPACE}/composable_kernel/build/CKInstallDir/lib/cmake/composable_kernel'
```

Comment:
Can we some how echo the commit here ? 

(Whilst I generally dont like ambiguous dependencies (i.e. CI not defining the commit to build), I felt the general consensus was to be able to check the latest -- thus, it would be great to know the commit of the last successful build)